.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: node_js
 node_js:
-  - "0.12"
+  - 6
+  - 4
```

Comment:
I agree. Let's only test with the most recent Node.js version. At https://docs.travis-ci.com/user/languages/javascript-with-nodejs I find:

<pre>
Use the following convenience versions:
- `node`: latest stable Node.js release
</pre>

Therefore I think it's best to use `node` instead of `6` to ensure that we always test using the latest stable version of Node.js.

After that, please squash the commits and this should be good to go!